Sources/GRPC/TLSVerificationHandler.swift

```diff
@@ -85,14 +85,12 @@ internal class TLSVerificationHandler: ChannelInboundHandler, RemovableChannelHa
         return
     }
 
-    self.logger.debug("TLS handshake completed, negotiated protocol: \(String(describing: negotiatedProtocol))")
     if let proto = negotiatedProtocol, GRPCApplicationProtocolIdentifier(rawValue: proto) != nil {
-      self.logger.debug("negotiated application protocol is valid")
-      self.verificationPromise.succeed(())
+      self.logger.debug("TLS handshake completed, negotiated protocol: \(proto)")
     } else {
-      self.logger.error("negotiated application protocol is invalid: \(String(describing: negotiatedProtocol))")
-      let error = GRPCError.client(.applicationLevelProtocolNegotiationFailed)
-      self.verificationPromise.fail(error)
+      self.logger.debug("TLS handshake completed, unknown protocol")
```

Comment:
nit: It's more like "no protocol negotiated" rather than unknown.

Reply:
Yep